modules/migration/review.go

```diff
@@ -18,6 +18,7 @@ const (
 	ReviewStateApproved         = "APPROVED"
 	ReviewStateChangesRequested = "CHANGES_REQUESTED"
 	ReviewStateCommented        = "COMMENTED"
+	ReviewStateRequestReview    = "REQUEST_REVIEW"
```

Comment:
Does this change really apply only to the gitea uploader, or does it extend to GitHub/ GitLab/ ... migrations as well?

Reply:
I don't know.

Reply:
Gitlab does not appear to have request reviews.

Github does and I have merged a commit that will add this.